sortinghat/cmd/unify.py

```diff
@@ -120,9 +127,18 @@ def unify(self, matching=None, sources=None,
         :param fast_matching: use the fast mode
         :param no_strict_matching: disable strict matching (i.e, well-formed email addresses)
         :param interactive: interactive mode for merging identities
+        :param log_path: path of the log containing matches
         """
         matcher = None
 
+        if not log_path:
```

Comment:
When I thought about adding this feature I thought of different behaviour:

* When a recovery or resume file is not given, the command works as always
* When a file is given:
    * If it doesn't exist, after the matching process, identities will be written to that file
    * If it exists, it means something went wrong the last time this process run, so it will skip matching process and will start merging identities.

What do you think? Don't you think is better this way?

Reply:
ok, I'll give it a try